src/amo/actions/reviews.js

```diff
@@ -43,7 +43,9 @@ export const UNLOAD_ADDON_REVIEWS: 'UNLOAD_ADDON_REVIEWS' =
   'UNLOAD_ADDON_REVIEWS';
 
 export type UserReviewType = {|
+  addonIconUrl: string,
```

Comment:
There is this issue about `icon_url` that should not be renamed to `iconUrl`: https://github.com/mozilla/addons-frontend/issues/2595. I guess it applies here too.

Reply:
I'm not sure I agree with #2595. Eslint will complain if we try to use variables with underscores in them, which I think is one of the reasons it was done this way in the first place. 

I'm not sure there will be a benefit in changing it back to `icon_url` and I fear it will introduce other hassles.

@kumar303 What do you think?

Reply:
> Eslint will complain if we try to use variables with underscores in them, which I think is one of the reasons it was done this way in the first place.

Yep, I am not sure about this issue either. It's good that you don't agree, I second that (although I understand why the issue has been filed).

Reply:
Well, maybe we should just close #2595 then. Too bad a contributor is already working on it. Any objections to that, @kumar303?

Reply:
alright, let's closing it.

Reply:
Calling this `addonIconUrl` seems fine to me. The issue you linked to has nothing to do with this 👻  It has to do with some scary code in Discopane that I think is long gone. I summarized it here: https://github.com/mozilla/addons-frontend/issues/2595#issuecomment-421492095